typings.d.ts

```diff
@@ -0,0 +1,308 @@
+declare module 'contentful-ui-extensions-sdk' {
+
+  interface Sys {
+    id: string;
+    type: string;
+    linkType?: string;
```

Comment:
there should be a separate interface `LinkSys` which includes `linkType`. `linkType` is only valid in the context of a link.